ansible/files/hetzner_server_nftables.conf

```diff
@@ -22,11 +24,35 @@ table inet filter {
   }
   chain FORWARD {
     type filter hook forward priority 0;
-    iifname virbr0 ip saddr 192.168.122.0/24 oifname virbr0 ip daddr 192.168.122.0/24 counter accept comment "allow all traffic between VMs"
+    iifname virbr0 oifname "enp*" counter jump block_definitions comment "no internet for internal VMs, disable this rule only for debugging"
```

Comment:
during experimentation with amit, we found we had to disable a rule, and write a new one, to have ip masquerading working here.